apps/web/src/components/Snackbar/__snapshots__/Snackbar.test.tsx.snap

```diff
@@ -12,47 +12,12 @@ exports[`the Snackbar component should render correctly with "error" variant 1`]
   onClose={[Function]}
   open={true}
 >
-  <div
-    className="makeStyles-container-1 makeStyles-error-5"
-  >
-    <div
-      className="makeStyles-contentContainer-2"
-    >
-      <div
-        className="makeStyles-iconContainer-3"
-      >
-        <ErrorIcon />
-      </div>
-      <div>
-        <WithStyles(ForwardRef(Typography))
-          className="makeStyles-headline-4"
-          component="span"
-          variant="body1"
-        >
-          Test Headline
-        </WithStyles(ForwardRef(Typography))>
-        <WithStyles(ForwardRef(Typography))
-          component="span"
-          variant="body1"
-        >
-           
-          Test Message
-        </WithStyles(ForwardRef(Typography))>
-      </div>
-    </div>
-    <div>
-      <WithStyles(ForwardRef(IconButton))
-        aria-label="close"
-        color="inherit"
-        onClick={[Function]}
-        size="small"
-      >
-        <Memo([object Object])
-          fontSize="small"
-        />
-      </WithStyles(ForwardRef(IconButton))>
-    </div>
-  </div>
+  <SnackbarImpl
```

Comment:
I don't think these snapshots are that useful. It's probably better to test the SnackbarImpl component itself, instead of the the Snackbar component which returns the SnackbarImpl component. This is the test file that I got working (I deleted two tests that don't seem relevant anymore):

```
import { shallow } from 'enzyme';
import React from 'react';
import { SnackbarImpl } from './Snackbar';

describe('the Snackbar component', () => {
  it('should render correctly with "warning" variant', () => {
    const wrapper = shallow(
      <SnackbarImpl variant="warning" headline="Test Headline" message="Test Message" handleClose={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly with "error" variant', () => {
    const wrapper = shallow(
      <SnackbarImpl variant="error" headline="Test Headline" message="Test Message" handleClose={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly with "info" variant', () => {
    const wrapper = shallow(
      <SnackbarImpl variant="info" headline="Test Headline" message="Test Message" handleClose={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly with no handleClose function provided', () => {
    const wrapper = shallow(<SnackbarImpl variant="error" headline="Test Headline" message="Test Message" />);
    expect(wrapper).toMatchSnapshot();
  });
});
```